src/js/panels.js

```diff
@@ -119,10 +119,162 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         that.locate("meter").css("height", percentage + "%");
     };
 
+    fluid.defaults("gpii.firstDiscovery.panel.keyboard", {
+        gradeNames: ["fluid.prefs.panel", "autoInit"],
+        preferenceMap: {
+            "gpii.firstDiscovery.stickyKeysEnabled": {
+                "model.stickyKeysEnabled": "default"
+            }
+        },
+        selectors: {
+            input: ".gpiic-fd-keyboard-input",
+            instructions: ".gpiic-fd-keyboard-instructions",
+            assistance: ".gpiic-fd-keyboard-assistance"
+        },
+        selectorsToIgnore: ["assistance"],
+        events: {
+            onOfferAssistance: null,
+            onInitInput: null
+        },
+        model: {
+            // offerAssistance: boolean
+            // tryAccommodation: boolean
+            userInput: ""
+        },
+        components: {
+            assistance: {
+                type: "gpii.firstDiscovery.keyboard.stickyKeysAdjuster",
+                createOnEvent: "onOfferAssistance",
+                container: "{that}.container",
+                options: {
+                    messageBase: "{keyboard}.options.messageBase",
+                    model: {
+                        tryAccommodation: "{keyboard}.model.tryAccommodation",
+                        stickyKeysEnabled: "{keyboard}.model.stickyKeysEnabled"
+                    }
+                }
+            },
+            stickyKeysAssessor: {
+                type: "gpii.firstDiscovery.keyboard.stickyKeysAssessment",
+                options: {
+                    requiredInput: "@",
+                    model: {
+                        userInput: "{keyboard}.model.userInput"
+                    },
+                    modelRelay: {
+                        source: "offerAssistance",
+                        target: "{keyboard}.model.offerAssistance",
+                        forward: "liveOnly",
```

Comment:
We should be able to get rid of this "liveOnly" and turn this into an implicit relay too - I don't see any dependence on initial model value